server/controllers/client.js

```diff
@@ -1,19 +1,74 @@
 const mongoose = require("mongoose");
+const axios = require("axios");
+const Result = require("../models/Result");
+const Report = require("../models/Report");
 
 module.exports = {
   getResult: async (req, res, next) => {
     const { resultId } = req.value.params;
+    const result = await Result.findById(resultId);
 
-    res.status(200);
+    if (!result) {
+      res.status(404).json({ error: { message: "Result not found" } });
+    } else {
+      res.status(200).json(result);
+    }
   },
 
   verify: async (req, res, next) => {
+    const { statement } = req.value.body;
+
+    try {
+      const result = await axios.post("http://127.0.0.1:8000/classify", {
+        statement: statement,
+      });
+      const fakeProbability = result.data.fake;
+      const trueProbability = result.data.true;
+      const verdict = trueProbability > fakeProbability ? "true" : false;
+      const probability =
+        trueProbability > fakeProbability ? trueProbability : fakeProbability;
+
+      const resultObject = await new Result({
+        statement: statement,
+        verdict: verdict,
+        probability: probability,
+      });
```

Comment:
Słówko kluczowe `await` jest zbędne przed `new Result({...`

W JS jeśli mamy `{ statement: statement }`, czyli nazwa pola pokrywa się ze zmienną, to można zapisać samo `{ statement }`

Całość mogłaby wyglądać tak:
```
      const resultObject = new Result({
        statement,
        verdict,
        probability,
      });
```